cdap-docs/developers-manual/source/pipelines/developing-plugins/presentation-plugins.rst

```diff
@@ -33,7 +39,9 @@ The widget JSON is composed of:
 
 - a map of :ref:`metadata <plugins-presentation-metadata>`
 - a list of property :ref:`configuration groups <plugins-presentation-configuration-groups>`
+- a map of :ref:`inputs  (input properties) <plugins-presentation-inputs>`
```

Comment:
These are not entirely a map of `input properties`. Since we are already talking about plugin properties mentioning properties here might be misleading.